tests/integration/test_provision_int.py

```diff
@@ -68,6 +68,31 @@ def test_provision_from_pyvenv(initproj, cmd, monkeypatch):
     assert ".tox/.tox/bin/python -m virtualenv" in result.out
 
 
+@pytest.mark.skipif(
+    "sys.version_info < (3, 7)",
+    reason="tox 4 only supports Python >= 3.7",
+)
+def test_provision_tox_4(initproj, cmd, monkeypatch):
+    initproj(
+        "pkg123-0.7",
+        filedefs={
+            "tox.ini": """\
+                [tox]
+                no_package=True
+                min_version = 4
+                [testenv]
+                commands=python -c "import os; print('assert this')"
```

Comment:
The import os part is a leftover from debugging a d should have been removed. Sorry about that.